lib/apivore/validator.rb

```diff
@@ -17,7 +17,12 @@ def initialize(method, path, expected_response_code, params = {})
     def matches?(swagger_checker)
       pre_checks(swagger_checker)
 
-      unless has_errors?
+      if has_errors?
```

Comment:
You might be right - I refactored it right before committing my change, and I might've made a mistake. I'll check it